### DISCUSSION TITLE
Very slow on 24GPU, any ideas why? :)

### DISCUSSION THREAD

**Discussion by patriciagomesoo** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#246] [Q&A] [answered]
Hello! I love your extension, for my watercolor artwork is amazing :) I just have one problem: when I'm using with controlnet tile on img2img tab, it gets soooo slow for upscaling a 1024x1536 image with a scale factor of 1.1 - 11minutes to upscale. When I do use UltimateSD it generates the same size with 3min only. I have a 24GPU card... maybe am I doing something wrong with settings? Does anyone have any ideas of what my be the problem? I do prefer Multi Diffusion results for my watercolor artwork than UltimateSD tho :)
My settings:
![Stable Diffusion(3)](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/69041628/2f3aa38b-9844-468b-bcd9-f897d9a0b54c)
![Stable Diffusion(4)](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/69041628/07100170-171b-4ca9-9533-388eb29d1745)
![Stable Diffusion(5)](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/69041628/9d978cf1-5054-46fc-9a33-874471aaf7e0)

Thank you :)

**Comment by pkuliyi2015** [comment, marked as answer]
The Tiled Diffusion overlap value is too large. Try 8 or 16 instead.

**Reply by patriciagomesoo** [reply]
Thank you!! It worked perfectly, it took just 2m24s, awesome, thank you :) I should have read your instructions before following some tutorial from civitai.
The colors are perfect, I love how the image looks, here's an example: [https://imgsli.com/MTg1NjA1](https://imgsli.com/MTg1NjA1)]
I know from seeing your examples, your final images are super sharp, any advice on parameter settings? In the example above I could not get a very sharp image, does the denoise inlfuence in some way sharpness? I have a very low denoise of 0.3. Sorry to bother you with so many questions, as I'm just starting out :)

**Reply by pkuliyi2015** [reply]
Your tiled diffusion tile size is too large to get sharper results as diffusion models tend to produce blur images when tiles are larger than 768. Normally speaking it should be 64 or 96 (corresponding to Ultimate Upscaler's 512 or 768). If other things are the same, the two methods will produce similar results.

**Reply by patriciagomesoo** [reply]
Thank you so much for answering and for sharing this amazing extension with the community :)
